lib/plugin.mock.js

```diff
@@ -19,10 +20,6 @@ function startPageTracking (ctx) {
 }
 
 export default function (ctx, inject) {
```

Comment:
What do you think to also add a notice about using mocked version?

Reply:
Sure

Reply:
But maybe better to have that log trigger only once from the module?

Reply:
I think would be better here because using browser devtools one can still see it is using mock version

Reply:
Done